src/pages/about.astro

```diff
@@ -103,12 +103,41 @@ const members = filterInactiveMembers(await getMembers());
       <section>
         <h2>What payments are eligible?</h2>
 
-        <p
-          >Companies directly pay maintainers of Open Source projects they depend on, with no strings attached. Paying
-          developers to do specific work does not count towards the Pledge. Feel free to participate in <a
-          href="https://github.com/opensourcepledge/osspledge.com/issues/112">our ongoing discussion</a> on what counts
-          and what doesn't.</p
-        >
+        <p>The primary intent with the Pledge is <b>no-strings-attached payments
+        to community projects</b>. Start there.</p>
+
+        <ol>
+
+          <li>Open Source projects that your company controls are out of scope.
+            If you own the trademark, it doesn't count.</li>
+
+          <li>Open Source projects that exclusively benefit your company or
+            your company's own ecosystem are out of scope.</li>
+
+          <li>It's okay if you get non-exclusive logo placement or other
+            acknowledgement for your payments, or an hour or two of consulting
+            time with the project team, but anything beyond such minimal
+            benefits puts the payment out of scope.</li>
+
+          <li>Payments to foundations are generally in scope, though most
+            foundations today are not set up to pay maintainers (they focus
+            more on conferences, trademarks, etc.). We hope that the Pledge
+            will enable foundations to shift towards paying maintainers in the
+            future.</li>
+
+          <li>It's great if your company also actively participates in the
+            projects you fund, via code contributions, membership on an
+            advisory council or board, and the like.</li>
+
+          <li>Platforms such as thanks.dev, Open Collective, and GitHub
+            Sponsors are a common way to pay maintainers, though they can also
+            be used to make payments that are out-of-scope for the Pledge.</li>
```

Comment:
thanks.dev can not be used to make payments though?

I'm also missing a "so be aware that just because you're using one of these platforms that doesn't necessarily mean that your donation qualify for the pledge"

Reply:
> thanks.dev can not be used to make payments though?

Sure it can! Maybe I'm misunderstanding?

Reply:
```suggestion
          <li>Platforms such as thanks.dev, Open Collective, and GitHub
            Sponsors are a common way to pay maintainers, though some can also
            be used to make payments that are out-of-scope for the Pledge. Only
            payments that fit our criteria are in scope, regardless of payment
            platform.</li>
```

Reply:
@vladh How? Its for bulk sponsorship? Bulk sponsorship is hard to use as payment system? Especially as you can't set up any tiers (its like Flattr in that regard)

Reply:
Oh, maybe I'm misunderstanding what you mean by “payment system”. thanks.dev is a way to pay maintainers — multiple maintainers at once, on a schedule, to be sure, but you can pay maintainers using it. Did you mean instant one-to-one payments by “payment system”?

Reply:
Paying = strings attached
Donating = no strings attached

In Open Collective and GitHub Sponsors you can attach strings to the transactions, turning donations into payments. (And eg. polar.sh specializes in adding such strings)

Reply:
> Paying = strings attached
> Donating = no strings attached

I don't like donating because it makes it sound like charity, which it's not. I would use "purchasing" for strings attached. We're inventing a new category so makes sense that language is unsettled yet. We should discuss elsewhere though: https://github.com/opensourcepledge/opensourcepledge.com/issues/173.

<img width="716" alt="image" src="https://github.com/user-attachments/assets/f57d8e60-35ac-4847-b247-5b5ba8a7f319">

Reply:
It was more in the context of this that I felt it was odd to mention thanks.dev:

> though some can also be used to make payments that are out-of-scope for the Pledge